### PR TITLE
Changed old .ix method to .loc/.iloc method #2004

### PR DESCRIPTION
**Pull request by garvpatidar04**
### :pencil: Description

**Type:** :beetle: `bugfix` | :rocket: `feature` | :biohazard: `breaking change` | :vertical_traffic_light: `testing` | :memo: `documentation` | :roller_coaster: `infrastructure`

Write a complete description of your changes, including the necessary context or any piece of information required to understand your work.

Also, link issues affected by this pull request by using the keywords: `close`, `closes`, `closed`, `fix`, `fixes`, `fixed`, `resolve`, `resolves` or `resolved`. 

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

**Comment by garvpatidar04** [comment]
> Hey! Thank you for making this PR. Please, make sure that the PR has a valid description.

@sonachitchyan 
Can you tell me what to do to make the mailmap check successful I have added my name and email in that file still it is not working?
And also what will be a valid description for this PR?
And can you also tell me what is 'codecov/patch — 0.00% of diff hit (target 67.34%)' as this is also failed?

Thank you

**Comment by andrewfullard** [comment]
> > Hey! Thank you for making this PR. Please, make sure that the PR has a valid description.
> 
> @sonachitchyan Can you tell me what to do to make the mailmap check successful I have added my name and email in that file still it is not working? And also what will be a valid description for this PR? And can you also tell me what is 'codecov/patch — 0.00% of diff hit (target 67.34%)' as this is also failed?
> 
> Thank you

Our mailmap bot seems to be a bit confused at the moment. A valid description will follow the guidelines in the default PR description and include what you did and why. Code coverage checks for test coverage in the code.

**Comment by garvpatidar04** [comment]
> > > Hey! Thank you for making this PR. Please, make sure that the PR has a valid description.
> > 
> > 
> > @sonachitchyan Can you tell me what to do to make the mailmap check successful I have added my name and email in that file still it is not working? And also what will be a valid description for this PR? And can you also tell me what is 'codecov/patch — 0.00% of diff hit (target 67.34%)' as this is also failed?
> > Thank you
> 
> Our mailmap bot seems to be a bit confused at the moment. A valid description will follow the guidelines in the default PR description and include what you did and why. Code coverage checks for test coverage in the code.

Ok, then I will change the description of PR

File: .mailmap [hunk 1]
```diff
@@ -223,3 +223,12 @@ Youssef Eweis <joey070@gmail.com> Youssef Eweis <Youssef15015@users.noreply.gith
 Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com>
 Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com> rohithvarma3000 <rohith.varma.buddaraju@gmail.com>
 Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com> Rohith <rohith.varma.buddaraju@gmail.com>
+
+GARV PATIDAR <patidargarv04@gmail.com> 
+garvpatidar04 <patidargarv04@gmail.com>
+GARV PATIDAR <patidargarv04@gmail.com> garvpatidar04 <patidargarv04@gmail.com>
+garvpatidar04 <notgarvpatidar@gmail.com>
+GARV PATIDAR <notgarvpatidar@gmail.com> garvpatidar04 <notgarvpatidar@gmail.com>
```
Comment (chvogl) [inline .mailmap:231]: You could try`GARV PATIDAR <90318877+garvpatidar04@users.noreply.github.com>` to see if it works.

**Review by chvogl — CHANGES_REQUESTED** [review]
The code changes look good to me but I agree with the previous reviewers that you should add a proper description for the PR. You did not fill in anything in the description template.

File: .mailmap (hunk 1 above)
Comment (garvpatidar04) [reply .mailmap:231]: Okay, I will try to do that.

**Comment by andrewfullard** [comment]
We should clean up the mailmap and just merge this.

**Pull request closed without merging**